Projects/UOContent/Items/Weapons/PoleArms/BasePoleArm.cs

```diff
@@ -6,17 +6,20 @@
 
 namespace Server.Items
 {
-    public abstract class BasePoleArm : BaseMeleeWeapon, IUsesRemaining
+    [Serializable(0)]
+    public abstract partial class BasePoleArm : BaseMeleeWeapon, IUsesRemaining
     {
-        private bool m_ShowUsesRemaining;
+        [SerializableField(0)]
+        [InvalidateProperties]
+        [SerializableFieldAttr("[CommandProperty(AccessLevel.GameMaster)]")]
+        private bool _showUsesRemaining;
 
-        private int m_UsesRemaining;
+        [SerializableField(1)]
+        [InvalidateProperties]
+        [SerializableFieldAttr("[CommandProperty(AccessLevel.GameMaster)]")]
+        private int _usesRemaining;
 
-        public BasePoleArm(int itemID) : base(itemID) => m_UsesRemaining = 150;
```

Comment:
This ctor was removed probably shouldn't have been.

Reply:
I think this is still present, it's just the diff made it look like it was removed when it was updated. See line 22 below:
```
public BasePoleArm(int itemID) : base(itemID) => _usesRemaining = 150;
```